src/BaseTransformer.ts

```diff
@@ -0,0 +1,46 @@
+import { transform, transformCollection } from './helpers/transformers';
+
+class BaseTransformer {
+    /**
+     * Perform the data mapping - this should be overridden by each specific Transformer
+     *
+     * @param data
+     * @param args
+     */
+    mapData(data: object, args: Array<any> = []): object {
+        return {
+            data,
+            ...args,
```

Comment:
Not certain we want to return args here. 

Essentially this is just an identity function so it should just return the data as is (as there's nothing to map it to)